File: pertemuan4/harga-barang-for/harga_for.c
#include <stdio.h>

int main() {
    int jumlah, i;
    float harga, total = 0;

    printf("Masukkan jumlah barang: ");
    scanf("%d", &jumlah);

    for (i = 1; i <= jumlah; i++) {
        printf("Masukkan harga barang ke-%d: ", i);
        scanf("%f", &harga);
        total += harga;
    }

    printf("Total harga: %.2f\n", total);
    return 0;
}

File: pertemuan4/harga-barang-for/harga_for.js
let jumlah = parseInt(prompt("Masukkan jumlah barang: "));
let total = 0;

for (let i = 1; i <= jumlah; i++) {
  let harga = parseFloat(prompt(`Masukkan harga barang ke-${i}: `));
  total += harga;
}

console.log(`Total harga: ${total.toFixed(2)}`);
